services/auth.js

```diff
@@ -138,6 +138,17 @@ const authService = {
     await User.updateOne({ _id: userId }, { $set: { password: hashedPassword } }).exec()
 
     await tokenService.removeResetToken(userId)
+  },
+
+  unsetFirstLogin: async (accessToken) => {
+    const tokenData = tokenService.validateAccessToken(accessToken)
```

Comment:
better remove this line and use authMiddleware

Reply:
fixed